sdk/cosmos/azure-cosmos/README.md

```diff
@@ -551,6 +551,39 @@ async def create_lists():
     item_list = [item async for item in results]
     await client.close()
 ```
+
+### Using Integrated Cache
+An integrated cache is an in-memory cache that helps you ensure manageable costs and low latency as your request volume grows. The integrated cache has two parts: an item cache for point reads and a query cache for queries. The code snippet below shows you how to use this feature with the point read and query cache methods.
+
+The benefit of using this is that the point reads and queries that hit the integrated cache won't use any RUs. This means you will have a much lower per-operation cost than reads from the backend.
+
+How to configure integrated cache on your Azure Cosmos DB account: [How to configure the Azure Cosmos DB integrated cache (Preview)][cosmos_configure_integrated_cache]
```

Comment:
I would just keep the link here - you have the same statement pointing at itself.